docs/lua/tables/gui.md

```diff
@@ -79,4 +79,30 @@ gui.show_error(title, message)
 bool = gui.is_open()
 ```
 
+### `add_imgui(imgui_rendering)`
+
+Registers a function that will be called every rendering frame, you can call ImGui functions in it, please check the ImGui.md documentation file for more info.
+**Example Usage:**
+```lua
+gui.add_imgui(function()
+   if ImGui.Begin("My Custom Window") then
+       if ImGui.Button("Label") then
+         script.run_in_fiber(function(script)
+           -- call natives in there
+         end)
+       end
+
+       ImGui.End()
+   end
+end)
+```
+
+- **Parameters:**
+  - `imgui_rendering` (function): Function that will be called every rendering frame, you can call ImGui functions in it, please check the ImGui.md documentation file for more info.
+
+**Example Usage:**
+```lua
+gui.add_imgui(imgui_rendering)
+```
+
```

Comment:
Also exists under `classes/tab.md`, is this intented?

Reply:
gui.

tab:

Not the same,though the descriptions could be clearer

Reply:
Oh yeah I see, in that way.
One draws to a Tab and the other is meant for raw drawing a UI with ImGui::Begin().